.golangci.yml

```diff
@@ -24,34 +22,29 @@ linters:
   disable:
     - cyclop            # covered by gocyclo
     - depguard          # unnecessary for small libraries
-    - deadcode          # abandoned
-    - exhaustivestruct  # replaced by exhaustruct
+    - execinquery       # deprecated since golangci v1.58
+    - exportloopref     # covered by copyloopvar
     - funlen            # rely on code review to limit function length
     - gocognit          # dubious "cognitive overhead" quantification
     - gofumpt           # prefer standard gofmt
     - goimports         # rely on gci instead
-    - golint            # deprecated by Go team
     - gomnd             # some unnamed constants are okay
-    - ifshort           # deprecated by author
-    - interfacer        # deprecated by author
+    - inamedparam       # convention is not followed
     - ireturn           # "accept interfaces, return structs" isn't ironclad
     - lll               # don't want hard limits for line length
     - maintidx          # covered by gocyclo
-    - maligned          # readability trumps efficient struct packing
+    - mnd               # status codes are clearer than constants
```

Comment:
Could use same comment as "gomnd" above, since they are the same linter:
```
# some unnamed constants are okay
```
(I am pretty sure that "gomnd" is being renamed to "mnd", and this is a version where both are active to support people migrating their config.)